pkg/network/proxy/proxy.go

```diff
@@ -332,12 +332,37 @@ func (proxy *OsdnProxy) OnEndpointsDelete(ep *corev1.Endpoints) {
 	}
 	delete(proxy.allEndpoints, ep.UID)
 	if !pep.blocked {
-		proxy.baseEndpointsHandler.OnEndpointsDelete(ep)
+		proxy.baseProxy.OnEndpointsDelete(ep)
 	}
 }
 
 func (proxy *OsdnProxy) OnEndpointsSynced() {
-	proxy.baseEndpointsHandler.OnEndpointsSynced()
+	proxy.baseProxy.OnEndpointsSynced()
+}
+
+func (proxy *OsdnProxy) OnServiceAdd(service *corev1.Service) {
+	klog.V(2).Infof("sdn proxy: add svc %s/%s: %v", service.Namespace, service.Name, service)
```

Comment:
@squeed is this a leftover?

Reply:
tried to fix & push over your branch here, but you haven't set that permission...

Reply:
It could be, but it's in master too. It doesn't do any harm, so let's just leave it.